assets/js/modules/analytics-4/datastore/audiences.test.js

```diff
@@ -304,6 +307,107 @@ describe( 'modules/analytics-4 audiences', () => {
 			} );
 		} );
 
+		describe( 'maybeSyncAvailableAudiences', () => {
+			it( 'should call syncAvailableAudiences if the availableAudiencesLastSyncedAt setting is undefined', async () => {
+				fetchMock.post( syncAvailableAudiencesEndpoint, {
```

Comment:
We tend to prefer `postOnce()` over `post()` where we can use it, as it's more specific and will fail if an unexpected post occurs:

```suggestion
				fetchMock.postOnce( syncAvailableAudiencesEndpoint, {
```